lib/global_registry/base.rb

```diff
@@ -23,6 +24,24 @@ def self.get(params = {}, headers = {})
     def get(params = {}, headers = {})
       request(:get, params, nil, headers)
     end
+    def self.get_all_pages(params = {}, headers = {})
```

Comment:
Ah, OK, that makes sense. So a `Base` class instance is effectively a "client" class that stores the token, url, and forwarding IP.

I was a bit confused by all the different subclasses that were for the different types of results and so I reached for adding helper methods to those, but now understanding that those don't do much (actually don't do anything at least right now) and that `Base` is the main class makes sense.